developers/ide/eclipse.md

```diff
@@ -5,65 +5,166 @@ title: Eclipse IDE
+    | openHAB Core Framework  | Core Framework Development            |
+
+    ::: warning
+    If you have selected `openHAB Add-ons` the installer will check out the openHAB2-addons repository  and all add-on projects are imported in Eclipse.
+    Select `OH2 Add-ons` and from right-click menu select "Close Projects": *this significantly speeds up the setup*.
+    Re-open only the binding project(s) you would like to work on.
+
+    If you want to develop a new binding or a specific binding it is recommended to clone your own fork of openHAB2-addons and import only the projects you work on.
+    :::
+
+
+1. Click `Next>`, verify/modify Root and install folder name. Click on `Show all variables` to open the window shown below.
+
+   ![variables](./images/ide_setup_eclipse_5_variables.png)
+
+   Explanation of some of the variables
+   - `Root install folder`: The base folder where the `Installation folder` will be placed.
+   - `Installation folder name`: This is the directory in the root install folder everything will be installed in.
+   - `GitHub user ID`: This is your GitHub user name used to configure the cloned Git projects.
+
+
+1. Click `Next>` and `Finish` to start installation.
+
+   During install accept licence agreement, "Unsigned Content" for Bndtools, and Eclipse Foundation certificates when requested to complete IDE installation.
+
+
+1. At this point the Eclipse installer is finished and the Eclipse IDE is automatically launched to continue the installation process.
+
+   **It's important**, during the first Eclipse IDE launch, to leave Eclipse open until all openHAB related initial Setup tasks / Download sources / Builds are completed.
 
     Setup tasks will personalize the IDE with openHAB code formatting tools, configurations and a demo app.
-	Setup tasks will also download openHAB lastest `openhab2-addons` and `openhab-distro` code repositories and related maven dependencies. 
+	Setup tasks will also download openHAB latest projects you have selected during installation. Like `openhab-distro` and the add-ons `openhab2-addons` project if you have selected it.
+
 	Click bottom right button in the IDE for Progress.
 
-    ![startup tasks progress](./images/ide_setup_eclipse_startup_tasks.png)
+    ::: warning
+    If you have selected `openHAB Add-ons` during the installation.
+    It may take 30-45mins for all initial tasks to finish, depending on internet connection and your computer speed.
+    So. Be aware of what you select during installation!.
+    And go and grab a good cup of coffee and be patient until is finished
+    :::
 
-    When visible in the Package Explorer, select `OH2 Add-ons` and from right-click menu select "Close Projects": this significantly speeds up the setup.
-    Re-open only the binding project(s) you would like to work on.
+   ![startup tasks finished](./images/ide_setup_eclipse_6_tasks_finished.png)
+
+1. After all tasks are finished you're ready to start developing.
```

Comment:
```suggestion
1. After all tasks are finished you are ready to start developing.
```

Reply:
The contraction is OK, but there should be a comma after `finished`.

Reply:
We moved in several review comments from `It's`to `it is`.
So i thought it would be a bit more consequent this way. :)

Reply:
Understood... it's more formal!